aperture.go

```diff
@@ -72,32 +72,32 @@ func Main() {
 	// TODO: Prevent from running twice.
 	err := run()
 	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
+		_, _ = fmt.Fprintln(os.Stderr, err)
```

Comment:
does this addition do anything?

Reply:
Just removes a linter warning in my IDE. I try to clean them up whenever I see them, mostly to reduce my code OCD symptoms :joy: